javascript/net/grpc/web/generator/grpc_generator.cc

```diff
@@ -1361,7 +1383,11 @@ void PrintGrpcWebClosureES6File(Printer* printer, const FileDescriptor* file) {
   string package_dot = file->package().empty() ? "" : file->package() + ".";
 
   printer->Print(
-      "// GENERATED CODE -- DO NOT EDIT!\n"
+      "// Code generated by protoc-gen-grpc-web. DO NOT EDIT.\n"
+      "// versions:\n"
+      "// \tprotoc-gen-grpc-web v$version$\n"
+      "// \tprotoc              v$protoc_version$\n"
+      "// source: $source_file$\n"
```

Comment:
oh sorry one more thing.. do you mind moving this block below the @fileoverview section for consistency? (i understand that this was the way before your change and there's probably nothing in the @fileoverview section that demands this change but probably good just for consistency and just in case we add more @params in the future which then it'll be broken :)) thanks!

Reply:
Fixed.

Reply:
Thanks again! :)